### PR TITLE
MNT: update setup.py for changes to codebase and python 3.7

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,13 +14,12 @@
...
     packages=setuptools.find_packages(),
-    package_data={'bluesky': ['schema/*.json']},
     scripts=glob.glob('scripts/*'),
     install_requires=requirements,
     classifiers=[
         "Development Status :: 3 - Alpha",
-        "Programming Language :: Python :: 3.4",
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
```
Comment (mrakitin) [inline setup.py]: Should we also add changes to PyPI description page, same as https://github.com/NSLS-II/ophyd/pull/583/files#r198996869?

**Comment by tacaswell** [comment]
zmq is failing to bind, only on 37 which is very odd...

```
E   zmq.error.ZMQError: Address already in use
```

**Comment by tacaswell** [comment]
I can reproduce this locally.

**Comment by tacaswell** [comment]
The ZMQ issues seem to have fixed them selves...

I'm going to cherry-pick this into #1125

**Pull request closed without merging**